Makefile

```diff
@@ -153,3 +156,23 @@ $(CONTROLLER_GEN): $(LOCALBIN)
 envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
 $(ENVTEST): $(LOCALBIN)
 	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+
+OSARCH=$(shell ./hack/get-os.sh)
+HELM = $(shell pwd)/bin/$(OSARCH)/helm
+HELM_INSTALLER ?= "https://get.helm.sh/helm-v3.10.1-$(OSARCH).tar.gz"
+.PHONY: helm
+helm: $(HELM) ## Download helm locally if necessary.
+$(HELM): $(LOCALBIN)
+	[ -e "$(HELM)" ] && rm -rf "$(HELM)" || true
+	cd $(LOCALBIN) && curl -s $(HELM_INSTALLER) | tar xvf - -C $(LOCALBIN) || tar xvf - -C $(LOCALBIN)
```

Comment:
This fails for me. Do you mean `tar xzf`? It seems like the file is gzipped, so I'd expect we run `gunzip`.

Reply:
This is actually specific to OS.. urgh! 
what are you running on?

Reply:
yes, we could use gunzip ill update

Reply:
> This is actually specific to OS.. urgh!
> what are you running on?

```
                   -`
                  .o+`
                 `ooo/                   todd@vixen 
                `+oooo:                  ---------- 
               `+oooooo:                 OS: Arch Linux x86_64 
               -+oooooo+:                Host: GS66 Stealth 10SE REV:1.0 
             `/:-:++oooo+:               Kernel: 5.19.12-arch1-1 
            `/++++/+++++++:              Uptime: 2 hours, 45 mins 
           `/++++++++++++++:             Packages: 1370 (pacman), 1 (flatpak) 
          `/+++ooooooooooooo/`           Shell: bash 5.1.16 
         ./ooosssso++osssssso+`          Resolution: 1080x1920, 1920x1080, 1920x1080 
        .oossssso-````/ossssss+`         DE: GNOME 42.5 
       -osssssso.      :ssssssso.        WM: Mutter 
      :osssssss/        osssso+++.       WM Theme: Adwaita 
     /ossssssss/        +ssssooo/-       Theme: Adwaita-dark [GTK2/3] 
   `/ossssso+/:-        -:/+osssso+-     Icons: Adwaita [GTK2/3] 
  `+sso+:-`                 `.-/+oso:    Terminal: vscode 
 `++:.                           `-/+/   CPU: Intel i7-10750H (12) @ 5.000GHz 
 .`                                 `/   GPU: NVIDIA GeForce RTX 2060 Mobile 
                                         GPU: Intel CometLake-H GT2 [UHD Graphics] 
                                         Memory: 11234MiB / 31919MiB 
```

BTW

Reply:
can you try this now? I think `tar was just missing the `-` flag prefix

Reply:
I get the same error:

```$ make helm
[ -e "/home/todd/git/open-feature-operator/bin/linux-amd64/helm" ] && rm -rf "/home/todd/git/open-feature-operator/bin/linux-amd64/helm" || true
cd /home/todd/git/open-feature-operator/bin && curl -s "https://get.helm.sh/helm-v3.10.1-linux-amd64.tar.gz" | tar -xvf - -C /home/todd/git/open-feature-operator/bin
tar: Archive is compressed. Use -z option
tar: Error is not recoverable: exiting now
make: *** [Makefile:167: /home/todd/git/open-feature-operator/bin/linux-amd64/helm] Error 2
```

`tar -xzf` does work for me (gnutar), but I'm not sure if that will break on bsdtar (which is I guess what you're using)

Reply:
that should work fine actually, I think the `c` is bsd specific